axelor-production/src/main/java/com/axelor/apps/production/service/BillOfMaterialServiceImpl.java

```diff
@@ -579,4 +582,32 @@ public List<BillOfMaterial> getSubBillOfMaterial(BillOfMaterial billOfMaterial)
 
     return new ArrayList<>();
   }
+
+  @Override
```

Comment:
Please return a Optional<BillOfMaterial>, it is even better since the method can return a null value